Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -148,7 +148,6 @@ class VLCMediaCategoryViewController: UICollectionViewController, UICollectionVi
         collectionView?.dataSource = editing ? editController : self
         collectionView?.delegate = editing ? editController : self
 
-        editController.resetSelections()
         displayEditToolbar()
```

Comment:
I believe that the logic of this method should be placed inside the new VC.

Reply:
That would mean that the editController needs to know about the MediaVCs view to perform it's display. Do you really want a subview reaching up to its superview? Shouldn't a view and it's viewcontroller rather control the position of its subviews?

Reply:
Well, now that EditController is a VC, I'm not sure why mediaCat should be the one taking care of updating the toolbar's constraint.

At least, if I understood correctly this would be what we have with this PR.

`MediaCatVC` --> `EditControllerVC` --> `EditToolbar`

So with this, I'm not sure that `MediaCatVC` should be the one updating/setting `EditToolbar`'s constraints.

Reply:
not quite, we have the editcontroller that is controlling the inside of the view but the edittoolbarview is a subview of the MediaCategory's view. What you propose is the case if we would push or present the EditControllerVC